app/build.gradle

```diff
@@ -259,6 +259,12 @@ task printBuildVariants {
     }
 }
 
+task printGeckoviewVersions {
+    doLast {
+        println "nightly: " + groovy.json.JsonOutput.toJson(GeckoVersions.nightly_version)
```

Comment:
Not relevant for this PR, but I wish there was a nicer way of moving data from `gradle` -> our automation without just scraping output with a delimiter!
Just discovered the [Gradle Tooling API](https://docs.gradle.org/current/userguide/embedding.html) :eyes:

:rotating_light: not an issue for this PR, but definitely interesting! :rotating_light: